sphinx/writers/latex.py

```diff
@@ -399,6 +401,15 @@ def __init__(self, document, builder):
         if self.elements['polyglossia']:
             self.elements['babel'] = ''  # disable babel forcely
             self.elements['multilingual'] = self.elements['polyglossia']
+            self.elements['classoptions'] += ',' + self.babel.get_language()
```

Comment:
If so, is it better to move this line to outside of the if-block?
The language option is defined only if babel or polyglossia is enabled. When both are disabled, the option is not defined (ex. Japanese docs).

Anyway, I understand this option is worthy. so you don't need to check how polygolossia works.